easy_my_coop/models/coop.py

```diff
@@ -97,7 +97,7 @@ def create_comp_sub_req(self, vals):
         vals["name"] = vals["company_name"]
         if not vals.get("partner_id"):
             cooperator = self.env["res.partner"].get_cooperator_from_crn(
-                vals.get("company_register_number")
+                vals.get("company_register_number", "")
```

Comment:
Rather than fixing here by passing a string, I'd make `def get_cooperator_from_crn` more robust and return False when passed a Falsy value ("", False, None).